airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -77,19 +78,48 @@ export function useDefaultTransformation(): OperationCreate {
 export const connectionValidationSchema = yup
   .object({
     name: yup.string().required("form.empty.error"),
-    scheduleType: yup.string().oneOf([ConnectionScheduleType.manual, ConnectionScheduleType.basic]),
-    scheduleData: yup
-      .object({
-        basicSchedule: yup
+    scheduleType: yup
+      .string()
+      .oneOf([ConnectionScheduleType.manual, ConnectionScheduleType.basic, ConnectionScheduleType.cron]),
+    scheduleData: yup.mixed().when("scheduleType", (scheduleType) => {
+      if (scheduleType === ConnectionScheduleType.basic) {
+        return yup.object({
+          basicSchedule: yup
+            .object({
+              units: yup.number().required("form.empty.error"),
+              timeUnit: yup.string().required("form.empty.error"),
+            })
+            .defined("form.empty.error"),
+        });
+      } else if (scheduleType === ConnectionScheduleType.manual) {
+        return yup.mixed().notRequired();
+      }
+
+      return yup.object({
+        cron: yup
           .object({
-            units: yup.number().required("form.empty.error"),
-            timeUnit: yup.string().required("form.empty.error"),
+            cronExpression: yup
+              .string()
+              .test({
+                name: "cronExpression",
+                message: "Incorrect cron string",
+                test(value) {
+                  if (value && isValidCron(value, { seconds: true, allowBlankDay: true })) {
+                    return true;
+                  }
+
+                  return this.createError({
+                    message: "form.empty.error",
```

Comment:
I don't think this should be `form.empty.error`, but rather some new error message. Ideally this error message would indicate why the cron string is invalid, but at the very least it should say something generic like "Invalid cron expression"